libraries/networking/src/EntityScriptUtils.h

```diff
@@ -11,11 +11,23 @@
 
 #ifndef hifi_EntityScriptUtils_h
 #define hifi_EntityScriptUtils_h
+#include <QMetaEnum>
 
-enum EntityScriptStatus {
-    ERROR_LOADING_SCRIPT,
-    ERROR_RUNNING_SCRIPT,
-    RUNNING
+class EntityScriptStatus_ : public QObject {
+    Q_OBJECT
+public:
+    enum EntityScriptStatus {
+        PENDING,
+        LOADING,
+        ERROR_LOADING_SCRIPT,
+        ERROR_RUNNING_SCRIPT,
+        RUNNING,
+        UNLOADED
+    };
+    Q_ENUM(EntityScriptStatus)
+    static QString valueToKey(EntityScriptStatus status) {
+        return QMetaEnum::fromType<EntityScriptStatus>().valueToKey(status);
+    }
```

Comment:
new statuses:
* `PENDING` == default state where an Entity script becomes well-known, but might get queued behind other still-loading Entities.
* `LOADING` == fetching with ScriptCache has begun
* `UNLOADED` == used to help troubleshoot earlier race conditions -- especially during "reload all."  (before the latest `ScriptEngines::stopAllScripts` logic was regularly observing "just deleted" Entities still getting scheduled for reloads)

Not entirely sure about using `QObject` + `Q_ENUM` + `QMetaEnum::fromType` here -- but it does save from having to maintain string versions of the enums by hand...  In this approach enums are not leaked into the global namespace, so do need to be fully-qualified in code (eg: `EntityScriptStatus::PENDING` instead of just `PENDING`).